File: 1.py
def to_bin(x):
    s = ''
    while(1):
        s += str(x%2)
        x //= 2
        if x < 1:
            break
    print(s[::-1])

def to_dec(X):
    n = 0
    for i in range(len(x)):
        n = n*2 + int(x[i])
    print(n)

x = input()
to_dec(x)

File: 1.cpp
#include <iostream>
#include <algorithm>

using namespace std;

int to_dec(string s){
    int n = 0;
    for (int i = 0; i < s.length(); i++){
        n = n*2 + (s[i] - '0');
    }

    return n;
}

string to_bin(int x){
    string s = "";
    while (x != 0){
        s += x%2 + '0';
        x /= 2;
    }
    reverse(s.begin(), s.end());
    return s;
}

int main(){
    

    return 0;
}